buffer/pom.xml

```diff
@@ -33,6 +33,13 @@
   </properties>
 
   <dependencies>
+    <dependency>
```

Comment:
If the new updated PR doesn't need substitutions, I believe this dependency can also go away right?

The remaining metadata is a plain property file so this isn't required.